repo/ecc/ecc.go

```diff
@@ -0,0 +1,87 @@
+// Package ecc implements common support for error correction in sharded blob providers
+package ecc
+
+import (
```

Comment:
reorder imports here based on linter failures

Reply:
I'm sorry for the question, but how can I see the linter output?

Reply:
`make lint` should do the trick. In general `make` before sending a PR will uncover all kinds of issues and save you time.

Reply:
Sorry. I had not noticed the Makefile there. All lint errors should be fixed now.